### DISCUSSION TITLE
Theme problem

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#561] [General]
why some app of kde apps not follow themes 
![Uploading 231126_19h52m33s_screenshot.png…]()

**Comment by kRHYME7** [comment]
please re upload the img.

**Reply by mahaveergurjar** [reply]
![231126_19h52m33s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/8883a286-69d7-43fa-8e54-63b505651653)

**Reply by mahaveergurjar** [reply]
![231127_08h11m24s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/8d2c0b9b-6824-4f1c-9e8b-8393bfabc136)

**Comment by mahaveergurjar** [comment]
![231127_08h15m32s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/4b022cdb-cf85-48d7-849d-2e817f56fdf6)

**Comment by mahaveergurjar** [comment]
Some kde plasma apps are not follow theme 
like kde Gnewview (photo app),  Okular( pdf reader app) , kde-connect and etc how to fix this issue
